tasks/images.js

```diff
@@ -22,6 +22,7 @@ gulp.task('images', callback => {
         'clean:images',
         ['copy:img', 'copy:assets'],
         'images:optimise',
+        ...config.docs.outputAssets ? ['copy:img:docs'] : [],
```

Comment:
What does the spread operator do here?

Reply:
Conditionally runs the `copy:img:docs` task if the `outputAssets` variable is true. We use the same technique in a few other placecs in this module.

Reply:
I'm confused about it too. Wouldn't the below do the same or am I missing something here ⚙️ 

`config.docs.outputAssets ? ['copy:img:docs'] : []`

Reply:
@kevinrodrigues The spread operator is actually being applied to the result of the ternary expression, so it returns either `'copy:img:docs'` or nothing.

Without it, either `['copy:img:docs']` or `[]` would be added to the list, which isn't what we want; passing `[]` into `runSequence` fails.

I asked Damian to put in some parentheses to help with the readability. [Here](http://2ality.com/2017/04/conditional-literal-entries.html)'s a good, short article on the same topic!